azure-pipelines.yml

```diff
@@ -0,0 +1,52 @@
+trigger:
```

Comment:
Is there a filename format that starts with a period?  like `.travis.yml`?

Reply:
I renamed the file to start with a `.`